data/sql/updates/pending_db_world/rev_1609174873784850500.sql

```diff
@@ -0,0 +1,29 @@
+-- INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609174873784850500');
+
+UPDATE `creature_template_addon` SET `auras` = '29266' WHERE (`entry` = 17508);
```

Comment:
```suggestion
UPDATE `creature_template_addon` SET `auras` = 29266 WHERE (`entry` = 17508);
```
Not a string

Reply:
It is a string.

Reply:
You combind auras by 'X Y Z' so this is correct.

Reply:
![image](https://user-images.githubusercontent.com/24550914/103240179-dcc3de00-494f-11eb-8181-012db00b2ee6.png)

Reply:
Thx for the information! Weird thing o.o